src/components/FusionAuthLoginButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
+import { useFusionAuthContext } from '../providers/FusionAuthProvider';
+
+interface Props {
+    redirectURI: string;
+    state: string;
```

Comment:
Make state optional here and default to empty string in login call if it is undefined